backend/de.metas.business/src/main/sql/postgresql/ddl/functions/getcreditlimitfordepartment.sql

```diff
@@ -0,0 +1,40 @@
+DROP FUNCTION IF EXISTS getcreditlimitfordepartment(p_m_department_id numeric,
+                                                    p_date            date)
+;
+
+CREATE OR REPLACE FUNCTION getcreditlimitfordepartment(p_m_department_id numeric,
+                                                       p_date            date) RETURNS numeric
+    STABLE
+    LANGUAGE sql
+AS
+$$
+SELECT COALESCE(SUM(lim.amount), 0)
+
+FROM C_BPartner partner
+         JOIN M_SectionCode sectionCode ON partner.m_sectioncode_id = sectionCode.m_sectioncode_id
+         JOIN m_department_sectioncode depSectionCode ON sectionCode.m_sectioncode_id = depSectionCode.m_sectioncode_id
+         JOIN M_Department dep ON depSectionCode.m_department_id = dep.m_department_id
+         JOIN C_BPartner_CreditLimit lim ON partner.c_bpartner_id = lim.c_bpartner_id
+
+
+WHERE dep.m_department_id = p_m_department_ID
+  AND ((lim.processed = 'Y' AND lim.isactive = 'Y') OR (lim.processed = 'N' AND lim.isactive = 'N'))
```

Comment:
what is this about? e.g. why ignore a credit-limit with `(lim.processed = 'N' AND lim.isactive = 'Y')`
please add an explanatory comment to this line

Reply:
extracted from : https://github.com/metasfresh/me03/issues/14325#issuecomment-1406979127
After the credit limit was deactivated, the credit limit usage is not modified because the deactivation was not approved.

Reply:
- sry i still don't get it. if there a connection between "approved" and `processed`? when comuting the creditlimit in the code, do we also have an analog to `((lim.processed = 'Y' AND lim.isactive = 'Y') OR (lim.processed = 'N' AND lim.isactive = 'N'))` in the whereclause?
- by to this line i meant the code-line, i.e. in the SQL. not in this comment that will soon valish in the distant past :-)

Reply:
> sry i still don't get it. if there a connection between "approved" and processed

In this context processed = approved, approved (technically processed='Y' )